catalog/content/core/content-core-impl/src/main/java/ddf/content/util/impl/DescribableImpl.java

```diff
@@ -13,11 +13,10 @@
  */
 package ddf.content.util.impl;
 
+import org.codice.ddf.platform.util.Describable;
```

Comment:
Never realized that we had abstract classes for this. We should deprecate those as well and have a common one. And I'm not sure we should keep the new one abstract.